eng/pipelines/templates/jobs/ci.yml

```diff
@@ -243,12 +239,14 @@ jobs:
               ServiceName: ${{parameters.ServiceDirectory}}
               ForRelease: false
 
+      - template: ../steps/generate-project-list.yml
```

Comment:
I need to remove this as it is now duplicated and I missed when I merged with the sparse checkout change.